### ISSUE TITLE
quickinstall.py does not work with ./quickinstall or similar

### ISSUE THREAD

**Issue by JanHoefelmeyer** [moinwiki/moin#1302]
Have not tested the issue on another platform yet.

On Windows 10, when trying to execute quickinstall, Windows will autocomplete it to './quickinstall.py', which is functionally identical to 'quickinstall.py'.

However, this does bypass the "if args == ['quickinstall.py']" clause to check whether it's a setup-install, causing the programm to throw an index error when trying to access args[1] in the last else clause: 
` Traceback (most recent call last):
  File "C:\Users\Arika\moin\quickinstall.py", line 763, in <module>
    choice = 'cmd_%s' % args[1]
IndexError: list index out of range`

**Comment by RogerHaase** [comment]
For the original installation and creation of the venv, you should do:

    <python> quickinstall.py

where <python> designates the python version you want to use. This is important if there are multiple versions of Python installed. If there is only one Python installed, then Windows will pick the right one, but the code fails on an IndexError.

This is probably fixable by a code enhancement, but then the docs will need to be made more complicated by saying do this if you multiple pythons installed, but if you have only one python then you may do it the first way but you may choose to do it this other way.

**Comment by JanHoefelmeyer** [comment]
Sorry, I worded myself poorly: I did do "python ./quickinstall.py", which did not work (with python being my python version). I omitted the python because I thought it was not of interest. 

The indexerror is because when the code checks on whether quickinstall has been called "alone" (line 743: if args == ['quickinstall.py']), the result is false if the input was '.\quickinstall.py' (-> which causes args to be ['.\quickinstall.py], although I would think technically it should be allowed to be called like this, since it's the correct file being called.

https://github.com/moinwiki/moin/pull/1303 might clarify what I mean.
